src/scip_data.jl

```diff
@@ -8,8 +8,9 @@ struct ConsRef
     val::Int64
 end
 
-"ManagedSCIP holds pointers to SCIP data and takes care of memory management."
-mutable struct ManagedSCIP
+#to be moved to MOI_wrapper
```

Comment:
Should this still be done, or this comment removed?

More consistent with the idea of this PR would be to move the content of `MOI_wrapper/` into the parent directory and eliminate many of the existing source files there, right?

Reply:
I think this was a comment from when I started to map out what should be moved where, since I thought it would be necessary. Since the current build works, I wouldn't move this to MOI_wrapper unless anybody with more insight than me wants me to (or to alternatively move the contents of MOI_wrspper/ into the parent dir).